File: 평강/app/project/src/main/java/com/eventpurchase/project/global/common/exception/ErrorCode.java
package com.eventpurchase.project.global.common.exception;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import org.springframework.http.HttpStatus;

@Getter
@RequiredArgsConstructor
public enum ErrorCode {
    PRODUCT_NOT_FOUND(HttpStatus.NOT_FOUND, "상품을 찾을 수 없습니다.");


    private final HttpStatus status;
    private final String message;


}

File: 평강/app/project/shared/src/main/kotlin/com/eventpurchase/project/shared/exception/ErrorCode.kt
package com.eventpurchase.project.shared.exception

enum class ErrorCode(val status: Int, val message: String) {
    PRODUCT_NOT_FOUND(404, "상품을 찾을 수 없습니다."),
    USER_NOT_FOUND(404, "유저를 찾을 수 없습니다."),
    CART_ITEM_NOT_FOUND(404, "장바구니 상품을 찾을 수 없습니다.")
}
